cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -4985,6 +5047,14 @@ macro(build_awssdk)
     set_property(TARGET CURL::libcurl
                  APPEND
                  PROPERTY INTERFACE_LINK_LIBRARIES OpenSSL::SSL)
```

Comment:
This is not related to this pull rquest but we should do this in `find_curl()`.

Reply:
Moved to find_curl